src/scripts/components/Actions.js

```diff
@@ -2,8 +2,59 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import helpers from '../helpers';
 
 
+/**
```

Comment:
This is largely a gut reaction (without thinking too much about it), but it seems weird for all of this stuff to be loaded just outside the component (lines 8-57) and not loaded elsewhere/independently.